app/assets/javascripts/components/reorderable-list.js

```diff
@@ -0,0 +1,84 @@
+//= require sortablejs/Sortable.js
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function ReorderableList () { }
+
+  ReorderableList.prototype.start = function ($module) {
+    this.$module = $module[0]
+    this.$upButtons = this.$module.querySelectorAll('.app-c-reorderable-list__actions button:first-of-type')
+    this.$downButtons = this.$module.querySelectorAll('.app-c-reorderable-list__actions button:last-of-type')
+
+    this.sortable = new window.Sortable.create(this.$module, { // eslint-disable-line new-cap
+      chosenClass: 'app-c-reorderable-list__item--chosen',
+      dragClass: 'app-c-reorderable-list__item--drag',
+      onSort: this.updateOrderIndexes.bind(this)
+    })
+
+    if (typeof window.matchMedia === 'function') {
+      this.setupResponsiveChecks()
+    } else {
+      this.sortable.option('disabled', true)
+    }
+
+    var boundOnClickUpButton = this.onClickUpButton.bind(this)
+    this.$upButtons.forEach(function (button) {
+      button.addEventListener('click', boundOnClickUpButton)
+    })
+
+    var boundOnClickDownButton = this.onClickDownButton.bind(this)
+    this.$downButtons.forEach(function (button) {
+      button.addEventListener('click', boundOnClickDownButton)
+    })
+  }
+
+  ReorderableList.prototype.setupResponsiveChecks = function () {
+    this.mql = window.matchMedia('(min-width: 40.0625em)')
+    this.mql.addListener(this.checkMode.bind(this))
+    this.checkMode()
+  }
+
+  ReorderableList.prototype.checkMode = function () {
+    if (this.mql.matches) {
+      this.sortable.option('disabled', false)
+    } else {
+      this.sortable.option('disabled', true)
```

Comment:
You can probably shorten this to `this.sortable.option('disabled', this.mql.matches)`

Reply:
Indeed, only the other way around (`!this.mediaQueryList.matches`) – I wasn't sure if the intent will remain clear. Let me know if you want me to shorten it like this.

Reply:
Replaced this logic with a one-liner